scripts/monitoring/cron-send-security-updates-count.py

```diff
@@ -0,0 +1,218 @@
+#!/usr/bin/python
+'''
+Send a count of pending security updates
+'''
+#   Copyright 2017 Red Hat Inc.
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# Purpose:
+# Report yum security update count to monitoring system
+
+from __future__ import print_function
+import argparse
+import re
+import shutil
+import fileinput
+import subprocess
+#pylint: disable=import-error
+from openshift_tools.monitoring.metric_sender import MetricSender
+from configobj import ConfigObj
+#pylint: enable=import-error
+import glob
+import yaml
+
+ROOT = '/var/local/hostpkg'
+LOGFILE = '/var/log/cron-send-security-updates-count.log'
+
+import logging
+
+class SecurityUpdates(object):
+    '''
+      This is a check to see how many security updates are pending
+    '''
+    def __init__(self, options):
+        '''SecurityUpdateCount constructor'''
+        self.verbose = options.verbose
+        self.sec_updates = []
+        self.sec_advisory_whitelist = []
+        wlfile = '/host/etc/openshift_tools/security_advisory_whitelist.yml'
+        try:
+            with open(wlfile, 'r') as yaml_file:
+                try:
+                    tmp = yaml.load(yaml_file)
+                    if 'advisories' in tmp:
+                        self.sec_advisory_whitelist = tmp['advisories']
+                        if self.verbose:
+                            print('Whitelisted advisories: {0}'.format(','.join(self.sec_advisory_whitelist)))
+                    elif self.verbose:
```

Comment:
Nope, this is correct. We only want to print the error if `'advisories' not in whitelist` (the `else` part) and `self.verbose` (the `if` part). Let me know if it should be rewritten to be less confusing.